### DISCUSSION TITLE
Nvim-dap-ui eval window focus issue

### DISCUSSION THREAD

**Discussion by ErnieBernie10** [LazyVim/LazyVim#2955] [Q&A] [answered]
Hi all. Lazyvim has worked amazingly for me so thanks a lot for that to the devs! There's one issue which I haven't been able to resolve. When I evaluate an argument with with dap a window appears. 

I want to focus this window, but whenever I move my cursor the window disappears. Anyone know how to actually enter this window or is this a bug?

![image](https://github.com/LazyVim/LazyVim/assets/30663198/7c48e1d3-621e-4519-8f82-cc2b34c0fa03)

Ideally it should autofocus this window which it currently does not do.

(with mouse mode enabled I can click inside the window and actually interact with it as expected. I just want to be able to do it with keyboard)

**Comment by dpetka2001** [comment, marked as answer]
Just press the same key again for evaluating with dap. You could make a custom keymap to overwrite the default like
```lua
keys = {
 {
	'<leader>de',
	function()
	  -- Calling this twice to open and jump into the window.
	  require('dapui').eval()
	  require('dapui').eval()
	end,
	desc = 'Evaluate expression',
	},
}
```

**Reply by ErnieBernie10** [reply]
That does the trick. Thanks!

**Reply by eerison** [reply]
Thank you <3
